src/taco-utils/testUtils/testHelper.ts

```diff
@@ -11,7 +11,11 @@
 
 "use strict";
 
-import should = require ("should");
```

Comment:
This one can stay as an import: since we directly refer to it in the rest of the file, the typescript compiler won't decide that it is unused.